### PR TITLE
Fix typos in Sources and Tests

### PR DESCRIPTION
**Pull request by Sajjon**
Fix some typos in `Sources` and `Tests`

Note: There is a complete mix of British vs American English spelling of (re/de)serialize vs serialise, but I refrain from changing those.

### CODE REVIEW

File: Sources/X509/X509BaseTypes/TimeCalculations.swift [hunk 1]
```diff
@@ -167,7 +167,7 @@ extension Int64 {
         // Same for remdays, the loop only terminates if the number is smaller than at most 31.
         //
         // Note that, unlike struct tm, we return ordinal month numbers as well as days (i.e. 1 to 12).
-        // This fits us better when working with generalizedtime and friends.
+        // This fits us better when working with generalized time and friends.
```
Comment (Sajjon) [inline Sources/X509/X509BaseTypes/TimeCalculations.swift:170]: I dont think `generalizedtime` is a reference to something, right? So this must be a typo...

Comment (Lukasa) [reply Sources/X509/X509BaseTypes/TimeCalculations.swift:170]: It is, that's the ASN1 type.

File: Sources/X509/Verifier/Verifier.swift [hunk 2]
```diff
@@ -33,7 +33,7 @@ public struct Verifier<Policy: VerifierPolicy> {
 
         var policyFailures: [VerificationResult.PolicyFailure] = []
 
-        // First check: does this leaf certificate contain critical extensions that are not satisfied by the policyset?
+        // First check: does this leaf certificate contain critical extensions that are not satisfied by the policy set?
```
Comment (Lukasa) [inline Sources/X509/Verifier/Verifier.swift:36]: `PolicySet` is a type.

File: Sources/X509/X509BaseTypes/TimeCalculations.swift (hunk 1 above)
Comment (dnadoba) [reply Sources/X509/X509BaseTypes/TimeCalculations.swift:170]: Probably should be uppercased though like this: [`GeneralizedTime`](https://en.wikipedia.org/wiki/GeneralizedTime)

**Review by Lukasa — CHANGES_REQUESTED** [review]

File: Sources/X509/X509BaseTypes/TimeCalculations.swift (hunk 1 above)
Comment (Sajjon) [reply Sources/X509/X509BaseTypes/TimeCalculations.swift]: fixed

File: Sources/X509/OCSP/OCSPPolicy.swift [hunk 3]
```diff
@@ -595,8 +595,8 @@ extension OCSPSingleResponse {
     ) -> PolicyEvaluationResult {
         /// Clients MUST check for the existence of the nextUpdate field and MUST
         /// ensure the current time, expressed in GMT time as described in
-        /// Section 2.2.4, falls between the thisUpdate and nextUpdate times.  Ifhttps://www.rfc-editor.org/rfc/rfc5019#section-4
-        /// the nextUpdate field is absent, the client MUST reject the response.
+        /// Section 2.2.4, falls between the thisUpdate and nextUpdate times.
+		/// If the nextUpdate field is absent, the client MUST reject the response.
```
[the remark this answers is not in the document]
Comment (Lukasa) [reply Sources/X509/OCSP/OCSPPolicy.swift:599]: This is still wrong.

[the remark this answers is not in the document]
Comment (Sajjon) [reply Sources/X509/OCSP/OCSPPolicy.swift]: Now! (I think) So spaces over tabs ey :)

**Comment by dnadoba** [comment]
@swift-server-bot test this please

**Review by Lukasa — APPROVED** [review]

**Pull request merged**